payments-core/src/main/java/com/stripe/android/payments/bankaccount/CollectBankAccountLauncher.kt

```diff
@@ -112,6 +112,20 @@ interface CollectBankAccountLauncher {
                 )
             )
         }
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun createForInstantDebits(
+            activityResultRegistryOwner: ActivityResultRegistryOwner,
+            callback: (CollectBankAccountResultInternal) -> Unit,
+        ): CollectBankAccountLauncher {
+            return CollectBankAccountForInstantDebitsLauncher(
+                activityResultRegistryOwner.activityResultRegistry.register(
+                    LAUNCHER_KEY,
+                    CollectBankAccountContract(),
+                    callback,
+                )
+            )
+        }
```

Comment:
feel free to just use the `CollectBankAccountForInstantDebitsLauncher` implementation directly

Reply:
Changed it! Will add the `callback` in https://github.com/stripe/stripe-android/pull/8299, because that’s the pull request containing the result handling code.